lib/trino-hdfs/src/main/java/io/trino/hdfs/authentication/HdfsKerberosConfig.java

```diff
@@ -41,10 +45,9 @@ public HdfsKerberosConfig setHdfsTrinoPrincipal(String hdfsTrinoPrincipal)
     }
 
     @NotNull
-    @FileExists
-    public String getHdfsTrinoKeytab()
+    public Optional<@FileExists String> getHdfsTrinoKeytab()
```

Comment:
Wow, I've not known that we can do that. So it's either null, or path to existing file correct?

Reply:
Yeah. And it applies check if the file exists.